src/libraries/System.Net.Http/tests/FunctionalTests/HttpContentTest.cs

```diff
@@ -829,6 +836,7 @@ public async Task ReadAsStreamAsync_Buffered_IgnoresCancellationToken(bool readS
         [InlineData(true)]
         [InlineData(false)]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/39187", TestPlatforms.Browser)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/42852", TestPlatforms.Browser)]
```

Comment:
#39187 seems to be a different issue? we should replace it with the one you added here

Reply:
Both of them possibly apply still.  I didn't try to resolve #39187.
There are at least 7 other places with 39187 on it.
I prefer to keep 39187 open until the whole test passes.